crates/bazed-core/src/app.rs

```diff
@@ -16,7 +16,7 @@ pub struct App {
 impl App {
     #[tracing::instrument(skip(self))]
     async fn load_file(&mut self, path: std::path::PathBuf) -> Result<()> {
```

Comment:
Should be some open_scratch or similar, probably document the temporary incorrect code.